cross/dante-sockd/Makefile

```diff
@@ -14,10 +14,4 @@ LICENSE = BSD/CMU
 GNU_CONFIGURE = 1
 CONFIGURE_ARGS = --disable-client --without-libwrap --without-bsdauth --without-gssapi --without-krb5 --without-upnp --without-pam
 
-PRE_CONFIGURE_TARGET = dante_pre_configure
-
 include ../../mk/spksrc.cross-cc.mk
-
-.PHONY: dante_pre_configure
-dante_pre_configure:
-	$(RUN) autoreconf -i -f -v
```

Comment:
This addition along with the patch allows it to build on DSM7.  Otherwise it does the following:
```
$ make arch-armv7-7.0
...
checking getaddrinfo() error symbols... configure: error: in `/home/spksrc/BKSteve/spksrc/cross/dante-sockd/work-armv7-7.0/dante-1.4.2':
configure: error: error: getaddrinfo() error value count too low
See `config.log' for more details
make[2]: *** [../../mk/spksrc.configure.mk:50: configure_target] Error 1
```

Reply:
That's very interesting and somewhat odd.

Reply:
Indeed but it's the only workaround I was able to find online...   Feel free to figure a better option, at least on master it now build ok with it.

Reply:
I don't have the patch for dante-sockd and I can't get that error message when building armv7-7.0 like you show.
If make armv7 then I get the cffi error and second build after spkclean is good.
I'm a little confused now.

Reply:
My guess is that your tree wasn't rebased against master as this update to version 1.4.3 is the last commit I pushed in and therefore your tree is removing this last commit on master.

The error I'm referring to is happening on all DSM7 builds and was caught on github-action from #4797.

Reply:
So you could just discard all the dante changes and use what's on master.

Reply:
Please all `dante` related changes as this is most probably due to your branch not being rebased against master as this was merged part of PR #4898 and yours removes it entirely and it isn't related to Sickchill at all.